src/platform/drivers/pl011_uart/inc/drivers/pl011_uart.h

```diff
@@ -1,210 +1,213 @@
-#define UART_ICR_OEIC            (1 << 10)
-
-/* UART DMA Control Register */
-
-#define UART_DMACR_RXDMAE        (1 << 0)
-#define UART_DMACR_TXDMAE        (1 << 1)
-#define UART_DMACR_DMAONERR      (1 << 2)
-
-/* For printk */
-
-#define serial_puts(str_buffer) uart_puts(1,str_buffer) 
-
-/* UART (PL011) register structure */
-
-struct Pl011_Uart_hw
-{
-   volatile uint32_t data;                // UART Data Register
-   volatile uint32_t status_error;        // UART Receive Status Register/Error Clear Register
-   const uint32_t reserved1[4];           	// Reserved: 4(0x4) bytes
-   volatile uint32_t flag;                // UART Flag Register
-   const uint32_t reserved2[1];            	// Reserved: 1(0x1) bytes
-   volatile uint32_t lp_counter;          // UART Low-power Counter Register
-   volatile uint32_t integer_br;          // UART Integer Baud Rate Register
-   volatile uint32_t fractional_br;       // UART Fractional Baud Rate Register
-   volatile uint32_t line_control;        // UART Line Control Register
-   volatile uint32_t control;             // UART Control Register
-   volatile uint32_t isr_fifo_level_sel;  // UART Interrupt FIFO level Select Register
-   volatile uint32_t isr_mask;            // UART Interrupt Mask Set/Clear Register
-   volatile uint32_t raw_isr_status;      // UART Raw Interrupt Status Register
-   volatile uint32_t masked_isr_status;   // UART Masked Interrupt Status Register
-   volatile uint32_t isr_clear;           // UART Interrupt Clear Register
-   volatile uint32_t DMA_control;         // UART DMA control Register
-};
-
-typedef struct Pl011_Uart_hw bao_uart_t;
-
-/** Public PL011 UART interfaces */
-
-void uart_disable(volatile struct Pl011_Uart_hw * ptr_uart);
-void uart_enable(volatile struct Pl011_Uart_hw * ptr_uart);
-void uart_set_baud_rate(volatile struct Pl011_Uart_hw * ptr_uart, uint32_t baud_rate);
-void uart_init(volatile struct Pl011_Uart_hw * ptr_uart);
-uint32_t uart_getc(volatile struct Pl011_Uart_hw * ptr_uart);
-void uart_putc(volatile struct Pl011_Uart_hw * ptr_uart,int8_t c);
-
-#endif /* __PL011_UART_H_ */
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) Bao Project and Contributors. All rights reserved.
+ */
+
+#ifndef __PL011_UART_H_
+#define __PL011_UART_H_
+
+#include <stdint.h>
+
+/* UART Base Address (PL011) */
+
+#define UART_BASE_0      0xFDF02000
+#define UART_BASE_1      0xFDF00000
+#define UART_BASE_2      0xFDF03000
+#define UART_BASE_4      0xFDF01000
+#define UART_BASE_5      0xFDF05000
+#define UART_BASE_6      0xFFF32000
+
+/* UART Interrupts */
+
+#define UART_0_INTERRUPT 106
+#define UART_1_INTERRUPT 107
+#define UART_2_INTERRUPT 108
+#define UART_4_INTERRUPT 109
+#define UART_5_INTERRUPT 110
+#define UART_6_INTERRUPT 111
+
+#define NUM_UART         6
+
+#define UART_CLK         19200000
+#define UART_BAUD_RATE   115200
+
+/* UART Data Register */
+
+#define UART_DATA_DATA   0xFFFFFF00
+#define UART_DATA_FE             (1 << 8
```

Comment:
missing ')' and not aligned with below macros

Reply:
Fixed.